tests/components/nordpool/conftest.py

```diff
@@ -51,25 +51,26 @@ async def load_int(
 
 @pytest.fixture(name="get_data")
 async def get_data_from_library(
-    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker, load_json: dict[str, Any]
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker, load_data: str
 ) -> DeliveryPeriodData:
     """Retrieve data from Nord Pool library."""
 
+    aioclient_mock.request(
+        "get",
+        API + "/DayAheadPrices",
+        text=load_data,
+        headers={"Content-Type": CONTENT_TYPE_JSON},
+        status=HTTPStatus.OK,
+    )
```

Comment:
This seems like a odd move, as this will put library testing on our end, which is generally not our goal.

Instead we should mock out the library itself, not the communication layer the library uses.

../Frenck

Reply:
Right, I think perhaps I took it one step to many. 👍